universum/__main__.py

```diff
@@ -52,7 +52,7 @@ def run(settings: ModuleNamespace) -> int:
     if error_state_module.is_in_error_state():
         raise IncorrectParameterError(("\n\n"+"-"*80 + "\n").join(error_state_module.get_errors()))
 
-    main_module.out.log("{} {} started execution".format(__title__, __version__))
+    main_module.out.log_execution_start(__title__, __version__)
```

Comment:
This call happens before artifacts cleaning.
And artifacts cleaning deletes output html file later.

Reply:
Actually, initial artifacts cleaning happens only in nonci mode or if "--clean-build" command line parameter is passed.